stories/why-step.stories.jsx

```diff
@@ -1,318 +1,234 @@
+//https://github.com/EnCiv/civil-pursuit/issues/103
+//https://github.com/EnCiv/civil-pursuit/issues/214
+
 import React, { useState, useEffect } from 'react'
-import { userEvent, within, waitFor } from '@storybook/test'
+import { userEvent, within, waitFor } from '@storybook/testing-library'
```

Comment:
testing-library isn't used any more.  It's just @storybook/test
and you can import expect from there too.
@storybook/jest is no longer there either.

Look at rerank-step.stories.jsx